android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -282,12 +284,54 @@ private boolean notReady(Promise promise) {
     }
 
     @SuppressWarnings("deprecation")
-    private int speak(String utterance, String utteranceId) {
+    private int speak(String utterance, String utteranceId, ReadableMap inputParams) {
+        int audioStreamType;
+
+        String audioStreamTypeString = inputParams.hasKey("KEY_PARAM_STREAM") ? inputParams.getString("KEY_PARAM_STREAM") : null;
+        float volume = inputParams.hasKey("KEY_PARAM_VOLUME") ? (float) inputParams.getDouble("KEY_PARAM_VOLUME") : (float) 1;
```

Comment:
Minor issue. 
- `1.0f` preferable to `(float) 1` (out of curiosity)?
- I wonder wasn't there any `getFloat()` method for this.

Reply:
- Thanks I didn't know about `1.0f` . It looks certainly better
- Yes, there was only `getDouble` for ReadableMap type